doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -0,0 +1,144 @@
+(rayjob-batch-inference-example)=
```

Comment:
```suggestion
(kuberay-batch-inference-example)=
```

You use `kuberay-batch-inference-example` in `examples.md` rather than `rayjob-batch-inference-example`.

Reply:
Thanks!